commit e788beaf093bfafecd6b4456b984bd927c18987a
Author: Joseph Myers <joseph@codesourcery.com>
Date:   Fri Apr 3 18:07:55 2020 +0000

    Update syscall lists for Linux 5.6.

    Linux 5.6 has new openat2 and pidfd_getfd syscalls.  This patch adds
    them to syscall-names.list and regenerates the arch-syscall.h files.

    Tested with build-many-glibcs.py.

Modified to only update syscall-names.list for RHEL 8.3.0.
Also cleaned up typos in the comments.

diff -Nrup a/sysdeps/unix/sysv/linux/syscall-names.list b/sysdeps/unix/sysv/linux/syscall-names.list
--- a/sysdeps/unix/sysv/linux/syscall-names.list	2020-04-06 12:27:10.519027866 -0400
+++ b/sysdeps/unix/sysv/linux/syscall-names.list	2020-04-06 12:31:53.665812810 -0400
@@ -16,14 +16,13 @@
 # License along with the GNU C Library; if not, see
 # <http://www.gnu.org/licenses/>.
 
-# This file contains the list of system call names names.  It has to
-# remain in alphabetica order.  Lines which start with # are treated
-# as comments.  This file can list all potential system calls.  The
-# names are only used if the installed kernel headers also provide
-# them.
+# This file contains the list of system call names.  It has to remain in
+# alphabetical order.  Lines which start with # are treated as comments.
+# This file can list all potential system calls.  The names are only
+# used if the installed kernel headers also provide them.
 
-# The list of system calls is current as of Linux 5.5.
-kernel 5.5
+# The list of system calls is current as of Linux 5.6.
+kernel 5.6
 
 FAST_atomic_update
 FAST_cmpxchg
@@ -293,6 +292,7 @@ open
 open_by_handle_at
 open_tree
 openat
+openat2
 osf_adjtime
 osf_afs_syscall
 osf_alt_plock
@@ -411,6 +411,7 @@ perf_event_open
 perfctr
 perfmonctl
 personality
+pidfd_getfd
 pidfd_open
 pidfd_send_signal
 pipe
